did/web/web.go

```diff
@@ -32,11 +32,19 @@ const (
 
 // Validate return nil if DID is valid, otherwise the validation error.
 func (d DIDWeb) Validate(ctx context.Context) error {
-	_, err := d.resolveDocBytes(ctx)
+	docBytes, header, err := d.resolveDocBytes(ctx)
 	if err != nil {
 		return errors.Wrap(err, "resolving doc bytes")
 	}
-	return nil
+	contentType := header.Get("Content-Type")
+	if contentType != "application/json" {
```

Comment:
extract the content type to a const, consider adding it to a util file
assuming there's no stdlib const

Reply:
Done.